packages/eas-cli/src/utils/statuspageService.ts

```diff
@@ -0,0 +1,53 @@
+import chalk from 'chalk';
+
+import {
+  StatuspageServiceFragment,
+  StatuspageServiceName,
+  StatuspageServiceStatus,
+} from '../graphql/generated';
+import { StatuspageServiceQuery } from '../graphql/queries/StatuspageServiceQuery';
+import Log, { link } from '../log';
+
+export async function maybeWarnAboutEasOutagesAsync(
+  serviceNames: StatuspageServiceName[]
+): Promise<void> {
+  const services = await getStatuspageServiceAsync(serviceNames);
+
+  for (const service of services) {
+    warnAboutServiceOutage(service);
+  }
+}
+
+function warnAboutServiceOutage(service: StatuspageServiceFragment): void {
+  const humanReadableServiceName: Record<StatuspageServiceName, string> = {
```

Comment:
can move this into module scope to avoid needing to create it each time the function is called